lib/uksched/include/uk/thread.h

```diff
@@ -141,18 +141,24 @@ struct uk_thread *uk_thread_current(void)
 #define UK_THREADF_UKTLS      (0x002)	/**< Unikraft allocated TLS */
 #define UK_THREADF_RUNNABLE   (0x004)
 #define UK_THREADF_EXITED     (0x008)
+#define UK_THREADF_QUEUEABLE  (0x010)
```

Comment:
For the following comments, I assumed that the meaning of the flag is "not running and not in the run-queue". It could be useful to incorporate the meaning in the documentation comment above.